packages/ecommerce-ui/components/FormComponents/ActionCardListForm.tsx

```diff
@@ -33,6 +35,7 @@ export function ActionCardListForm<
     multiple,
     disabled,
     shouldUnregister,
+    forceSelection,
```

Comment:
I personally like the 'disableXY' naming convention, so something like:`disableDeselectOption`? What do you think?